docs/content/guides/testing-assets.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Testing Assets | Dagster
+description: Dagster enables you to unit-test individual assets and graphs of assets
+---
+
+# Testing assets
+
+Creating testable and verifiable data pipelines is one of the focuses of Dagster. We believe ensuring data quality is critical for managing the complexity of data systems. Here, we'll cover how to write unit tests for individual assets, as well as for graphs of assets together.
+
+Let's go back to the assets we defined in the [tutorial](/tutorial) and ensure that they work as expected by writing some unit tests.
```

Comment:
🔧 It also sounds like the user should have completed the tutorial before starting this guide. If that's the case, this should have a **Prerequisites** section that says that.